src/createComponent.js

```diff
@@ -16,6 +16,10 @@ import React from 'react';
  * @public
  */
 export default function createComponent(options = {}) {
+  if (!options.render || typeof options.render !== 'function') {
+    throw new Error('Render is a required method.');
```

Comment:
I would suggest that the error message is changed to use `render` with lowercase letters, for instance: "Component missing required method: 'render'".  The name of the method appearing in the error message with capital letter can be misleading.  But other than that, very nice job!